### ISSUE TITLE
Suggestion: Change "practical chance"

### ISSUE THREAD

**Issue by anseljh** [rxnda/rxnda-forms#4]
In 4(g) of B-1W-B2I, there is this sentence:

> If practical, Receiving Party shall give notice quickly enough to afford Disclosing Party practical chance to start a proceeding to protect the confidentiality of the Confidential Information.

Consider changing "practical chance" to "practical opportunity". You're going to court, not the casino. :)

Feel free to disregard if *MSCD* says otherwise, or if you think it's too wordy, etc.

**Comment by kemitchell** [comment]
This language is admittedly loose. I'm not sure that's a bad thing, but I'm no litigator, so I'm sure I lack the know-how to make it more specific, while still making sense in any US jurisdiction, if indeed that's possible.

Might it speak in terms of a specific filing or other deadline? Perhaps with some hard-line guess as to the actual time needed for preparation, say "at least thirty calendar days before..."?

**Comment by anseljh** [comment]
Broad seems OK here. I don't think you should try to anticipate every scenario.

On further thought, I'd go a little broader and delete "start a proceeding to", because your first protective step is probably something other than starting a proceeding.

> If practical, Receiving Party shall give notice quickly enough to afford Disclosing Party a practical opportunity to protect the confidentiality of the Confidential Information.

For some context, here's what [Google says](https://www.google.com/transparencyreport/userdatarequests/legalprocess/) about how much time a user has to respond to their notice of a law enforcement request:

> **What can I do about a request like this?**
> You may wish to consult a lawyer to discuss your options. In our notice to you, we will provide information so that you can contact the requesting party with questions about the legal process. We will also provide a copy of the legal process upon request, although we may have to redact some information before sending it to you. We can’t give you legal advice or discuss the substance of the request.
> Unless you take action, like filing an objection with the court, we may have to produce information responsive to the request. Typically, the amount of time you’ll have to file an objection will be 7 calendar days, though that can vary from case to case. Be sure to send us a copy of any objection filed with the court so we know about it, and make sure that copy has the court’s stamp on it showing it was actually filed. It is not enough to just ask us not to disclose the information because we may be required to produce data unless a court tells us otherwise.

**Comment by kemitchell** [comment]
I'm glad to hear this can stay short and somewhat broad. I'd like to put just a slightly sharper point of your proposal, however. What do you think of?:

> If practical, Receiving Party shall give notice quickly enough to afford Disclosing Party a practical opportunity __to take legal action to__ protect the confidentiality of the Confidential Information.

**Comment by anseljh** [comment]
Better. Now I'm looking at "If practical" and feeling a little queasy about that. Maybe it's giving too much latitude to the Receiving Party, which can just say "nope, not practical enough, too bad"?

**Comment by kemitchell** [comment]
How about "if possible"?

Off the cuff here: If through no fault of the receiving party, there's just no time to start legal action within deadline, even if notice came immediately, there shouldn't be any argument for breach.

**Comment by anseljh** [comment]
I think I like "if possible".

**Comment by kemitchell** [comment]
New language will go out in `1e1u`s shortly.
